src/107-Arduino-TMF8801.cpp

```diff
@@ -80,14 +80,18 @@ bool ArduinoTMF8801::begin(uint8_t const measurement_period_ms)
   _api.clearInterrupt (TMF8801::InterruptSource::ObjectDectectionAvailable);
   _api.enableInterrupt(TMF8801::InterruptSource::ObjectDectectionAvailable);
 
+  /* set gpio_control to Input for GPIO0 and GPIO1
+   */
+  gpio_control=0x00;
```

Comment:
It's better to initialize member variables with default values in the Constructor, please move initialisation up there.

Reply:
should be fixed now.